tox.ini

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+[tox]
+requires =
+    tox>=4
+envlist = make-prs
+
+[testenv]
+basepython = python3
+setenv =
+  PY_COLORS=1
+passenv =
+  GITHUB_TOKEN
+
+[testenv:make-prs]
+description = Build the Sphinx docs
+deps = pip-tools
```

Comment:
Is this actually required? I can't see where it's being used, but I might be missing it.